src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -17,18 +17,28 @@ object Lorre extends App with LazyLogging {
 
   private val conf = ConfigFactory.load
   private val awaitTimeInSeconds = conf.getInt("dbAwaitTimeInSeconds")
-  val sleepIntervalInSeconds = conf.getInt("lorre.sleepIntervalInSeconds")
+  private val sleepIntervalInSeconds = conf.getInt("lorre.sleepIntervalInSeconds")
+  private val feeUpdateInterval = conf.getInt("lorre.feeUpdateInterval")
 
   lazy val db = DatabaseUtil.db
   val tezosNodeOperator = new TezosNodeOperator(TezosNodeInterface)
 
+  var iterationsOfLorre = 0
+  var shouldFeeBeCalculatedThisIteration = iterationsOfLorre == 0 || iterationsOfLorre % feeUpdateInterval == 0
+
   try {
     while(true) {
       logger.info("Fetching blocks")
       processTezosBlocks()
       logger.info("Fetching accounts")
       processTezosAccounts()
+      if (shouldFeeBeCalculatedThisIteration) {
+        logger.info("Fetching fees")
+        FeeOperations.processTezosAverageFees()
+      }
       logger.info("Taking a nap")
+      iterationsOfLorre = iterationsOfLorre + 1
+      shouldFeeBeCalculatedThisIteration = iterationsOfLorre == 0 || iterationsOfLorre % feeUpdateInterval == 0
```

Comment:
Can this line be removed?

Reply:
This was the bug, the boolean was a val, not a var, so it wasn't updating each loop. It needs to be updated, whether it's a variable, or just a condition in the if statement

Reply:
So can we set the initial value on line 27 to simply `true`? Remember, DRY.

Reply:
Got it!